неверный вывод ошибок в шаблонах include (не определен класс)

{from &i = 10 to 20 step 1}

"true" => true

внедрить {ifempty} внутрь FOREACH

косяк с let, когда он находится вне тегов

запилить геттеры и сеттеры

Controllers.get()

**~var = (!!~var?~var:'')

User.aaa()

Dictionary.aaa()

5*#data -> error

5*#data['aaa'] -> not error

$func(), ~func()